examples/hooks/main.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	hplugin "github.com/hashicorp/go-plugin"
+
+	"hooks/cmd"
+
+	"github.com/ignite/cli/v28/ignite/services/plugin"
+)
+
+type app struct{}
+
+func (app) Manifest(_ context.Context) (*plugin.Manifest, error) {
+	return &plugin.Manifest{
+		Name: "hooks",
+		Hooks: []*plugin.Hook{
+			{
+				Name:        "chain-build",
+				PlaceHookOn: "ignite chain build",
+			},
+			{
+				Name:        "chain-serve",
+				PlaceHookOn: "ignite chain serve",
+			},
+		},
+		Commands: cmd.GetCommands(),
+	}, nil
+}
+
+func (app) Execute(ctx context.Context, c *plugin.ExecutedCommand, _ plugin.ClientAPI) error {
+	fmt.Println("To use either run \"ignite chain build\" or \"ignite chain serve\" and see the output.")
```

Comment:
It would be also possible to use "`" to avoid escaping here and also in the string down:

```suggestion
	fmt.Println(`To use either run "ignite chain build" or "ignite chain serve" and see the output.`)
```